packages/perseus/src/__tests__/renderer_test.jsx

```diff
@@ -305,21 +303,23 @@ describe("renderer", () => {
             });
         });
 
-        beforeEach(() => {
-            // Reset "known" images
-            images.splice(0, images.length);
-        });
-
-        afterAll(() => {
+        afterEach(() => {
             window.Image = originalImage;
         });
```

Comment:
`beforeAll` and `afterAll` isn't a good choice for this since they don't run reliably when using wallaby to solo individual tests.